browser/pages/location/controller.js

```diff
@@ -40,6 +42,23 @@ class LocationController {
     this.checkFolder();
   }
 
+  checkUserProfilestatus() {
+    return new Promise((resolve, reject)=> {
+      if (Platform.OS === 'win32') {
+        Util.executeCommand('echo %USERPROFILE%',1).then((path)=>{
```

Comment:
should check for %TMP% value instead in case it is set directly in control panel.